k8s/pkg/apis/longhorn/v1beta2/orphan.go

```diff
@@ -20,6 +23,9 @@ const (
 )
 
 const (
+	OrphanInstanceName   = "InstanceName"
+	OrphanDataEngineType = "DataEngineType"
```

Comment:
Could we add a field `DataEngine` in OrphanSpec instead?

Reply:
Add new field `DataEngine` in `OrphanSpec`, and the webhook validate when the orphan type is instance orphans.